Magento_Theme/layout/default.xml

```diff
@@ -166,14 +141,18 @@
 
         <!-- End of Footer layout -->
 
-
-
         <!--Removed blocks-->
         <referenceBlock name="report.bugs" remove="true"/>
         <referenceBlock name="navigation.sections" remove="true" />
 
+        <!--Removed constainers-->
+        <referenceContainer name="header.container" remove="true" />
+
         <!--Moved blocks-->
-        <move element="skip_to_content" destination="header.container" before="-"/>
```

Comment:
on the beginning on the header we should add "Skip to content" link. (it's not visible on the website now). it should looks like on Luma theme and our mixin `visually-hidden` can be used here but visibility on focus should be added. If you'll have a question, let me know on chat, it was implemented in old version of alpaca header ;)

Reply:
Added new issue https://github.com/SnowdogApps/magento2-alpaca-components/issues/154